cmd/oras/cp.go

```diff
@@ -117,6 +120,9 @@ func runCopy(opts copyOptions) error {
 	committed := &sync.Map{}
 	extendedCopyOptions := oras.DefaultExtendedCopyOptions
 	extendedCopyOptions.Concurrency = opts.concurrency
+	extendedCopyOptions.FindPredecessors = func(ctx context.Context, src content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		return graph.Referrers(ctx, src, desc, "")
```

Comment:
Using `graph.Referrers` is workable but not efficient as we don't need to fetch the manifest for artifact types and annotations, which we don't care here.

Reply:
Shouldn't remote repository always be a `ReferrerLister` and don't need to enrich descriptor?